extensions/amp-story/1.0/amp-story-open-page-attachment.css

```diff
@@ -234,3 +236,71 @@
   margin-bottom: 20px !important;
   color: var(--i-amphtml-label-text-color) !important;
 }
+
+/** For amp-story-outlink-page-attachment-v2 experiment elements. */
+.i-amphtml-story-outlink-page-attachment-arrow {
+  display: block !important;
+  height: 32px !important;
+  width: 32px !important;
+  cursor: pointer !important;
+  position: relative;
+}
+
+.i-amphtml-story-outlink-page-open-attachment-bar-left {
+  background: white !important;
+  left: 6px !important;
+  animation: open-attachment-bar-left 0.3s cubic-bezier(0.4, 0.0, 0.2, 1) both !important;
+}
+
+.i-amphtml-story-outlink-page-open-attachment-bar-right {
+  background: white !important;
+  right: 6px !important;
+  animation: open-attachment-bar-right 0.3s cubic-bezier(0.4, 0.0, 0.2, 1) both !important;
+}
+
+.i-amphtml-story-outlink-page-attachment-outlink-chip {
+  display: flex !important;
+  position: relative !important;
+  padding: 10px 14px 10px 6px !important;
+  margin: 0 0 20px 0 !important;
+  height: 16px !important;
+  max-width: 243px !important;
+  border-radius: 30px !important;
+  background: white !important;
+  color: var(--cta-accent-color, black) !important;
+  place-items: center !important;
+}
+
+.i-amphtml-story-outlink-page-attachment-img {
+  height: 24px !important;
+  width: 24px !important;
+  vertical-align: middle !important;
+  background-size: contain !important;
+  background-repeat: no-repeat !important;
+  background-position: center !important;
+  border-radius: 50% !important;
+  background-image: url('data:image/svg+xml;charset=utf-8,<svg fill="none" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><rect width="24" height="24" rx="12" fill="#E8EAED"/><path d="M10 18s0 0 0 0l2-1 2-2s0 0 0 0h-1l-2 1a3 3 0 01-3 0 3 3 0 010-3l1-2v-1s0 0 0 0l-2 2a4 4 0 000 5l3 1zM11 9l2-1a3 3 0 013 0 3 3 0 010 3l-1 2v1a1 1 0 000 0l2-2a4 4 0 000-5l-3-1s0 0 0 0l-2 1-2 2s0 0 0 0h1z" fill="#000" stroke="#000" stroke-width=".3"/><path d="M15 9h-1l-5 5a1 1 0 001 1s0 0 0 0l5-5V9z" fill="#000" stroke="#000" stroke-width=".3"/></svg>') !important;
+}
+
+.i-amphtml-story-outlink-page-attachment-label {
+  font-family: 'Roboto', sans-serif !important;
+  font-size: 16px !important;
+  font-weight: bold !important;
+  letter-spacing: 0.3px;
+  line-height: 16px !important;
+  overflow: hidden !important;
+  text-overflow: ellipsis !important;
+  white-space: nowrap !important;
+  max-width: 210px !important;
+  vertical-align: middle;
+  padding-left: 6px !important;
+}
+
+.i-amphtml-story-outlink-page-attachment-outlink-chip-no-image {
+  padding-left: 14px !important;
```

Comment:
You have a few padding-left properties in your code that will break RTL layouts. Can you either use flexboxes or use padding-inline-start, and manually test this feature for RTL layouts?

Reply:
Fixed all the padding with padding-inline-start & padding-inline-end. Here's the RTL manual test:

![RTL](https://user-images.githubusercontent.com/10503669/114066449-bfb17c00-9869-11eb-9c16-d3c4c677b321.gif)